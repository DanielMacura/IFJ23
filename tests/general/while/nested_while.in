var i = 0 ;
while( i < 2 ) {
    var j = 0 ;
    var a = 1 ;
    i = i + a ;
    while( j < 2 ) {
        j = j + i
        let i = 9
        write(i, j, "\n")
    }
    write(i, j, "\n")
}