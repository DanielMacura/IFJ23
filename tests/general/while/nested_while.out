91
92
12
92
22
